include/swift/Option/FrontendOptions.td

```diff
@@ -33,6 +33,9 @@ def supplementary_output_file_map : Separate<["-"], "supplementary-output-file-m
 def frontend_parseable_output : Flag<["-"], "frontend-parseable-output">,
   HelpText<"Emit textual output in a parseable format">;
 
+def access_notes : Separate<["-"], "access-notes">, MetaVarName<"<path>">,
```

Comment:
This should probably be `-access-notes-path` or something to match the flags for other compiler inputs. I think there might also be a way I should be marking this flag as having a path argument?